test/spec/title.js

```diff
@@ -0,0 +1,13 @@
+var assert = require('assert');
+
+describe('page title', function() {
+
+  it('should contain the words "College Scorecard"', function*() {
+
+    yield browser.url('/');
+    var title = yield browser.getTitle();
```

Comment:
'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'browser' is not defined.